src/KomaUI.jl

```diff
@@ -113,7 +113,6 @@ global recParams = merge(default, rec)
 default = Dict{String,Any}()
 global simParams = merge(default, sim)
 #GPUs
-@info "Loading GPUs"
```

Comment:
I think the info message is ok. Do not delete it!

Reply:
Edited in commit [9079e70](https://github.com/cncastillo/KomaMRI.jl/commit/9079e706da5dff875166b6ae0e8cc17f3872f68d).